src/x11_window.c

```diff
@@ -1130,6 +1130,14 @@ static void releaseMonitor(_GLFWwindow* window)
     }
 }
 
+static void _glfwInputCharX11(_GLFWwindow* window, unsigned int codepoint, int mods, GLFWbool plain)
```

Comment:
You should rename this function `inputChar()` or something similar, the `_glfwInput` prefix is reserved for message passing from the backend to the core of the library.